core/src/main/java/org/apache/lucene/index/OneMergeHelper.java

```diff
@@ -27,4 +29,31 @@ private OneMergeHelper() {}
     public static String getSegmentName(MergePolicy.OneMerge merge) {
         return merge.info != null ? merge.info.info.name : "_na_";
     }
+
+    /**
+     * The current MB per second rate limit for this merge.
+     **/
+    public static double getMbPerSec(Thread thread, MergePolicy.OneMerge merge) {
+        if (thread instanceof ConcurrentMergeScheduler.MergeThread) {
+            return ((ConcurrentMergeScheduler.MergeThread) thread).rateLimiter.getMBPerSec();
+        }
+        return Double.POSITIVE_INFINITY;
```

Comment:
should we `assert false` here to get test failures if the thread is not a merge thread?